src/current/_includes/releases/v24.1/v24.1.3.md

```diff
@@ -0,0 +1,71 @@
+- Fixed a bug where a [user-defined function (UDF)]({% link v24.1/user-defined-functions.md %}) that shared a name with a [built-in function]({% link v24.1/functions-and-operators.md %}#built-in-functions) would not be resolved, even if the UDF had higher precedence according to the [`search_path`]({% link v24.1/sql-name-resolution.md %}#search-path) variable. [#126296][#126296]
+- Fixed a bug that caused [background jobs]({% link v24.1/show-jobs.md %}) to incorrectly respect a statement timeout.  [#126820][#126820]
+- Fixed a bug where [`ALTER DATABASE ... DROP REGION`]({% link v24.1/alter-database.md %}#drop-region) could fail if any tables under the given database have [indexes on expressions]({% link v24.1/expression-indexes.md %}). [#126599][#126599]
+- Fixed a PostgreSQL incompatibility bug when inputting `public` as user name for [built-in functions]({% link v24.1/functions-and-operators.md %}) such as `has_database_privilege` and `has_schema_privilege`. [#126851][#126851]
+- Fixed a bug when [restoring]({% link v24.1/restore.md %}) a database with a [composite type]({% link v24.1/create-type.md %}#create-a-composite-data-type). [#126428][#126428]
+- Fixed a bug where the [Databases]({% link v24.1/ui-databases-page.md %}) page crashed if the range information was not available. [#127092][#127092]
+- Fixed a bug where CockroachDB could incorrectly evaluate an [`IS NOT NULL`]({% link v24.1/null-handling.md %}#nulls-and-simple-comparisons) filter if it was applied to non-`NULL` tuples that had `NULL` elements, such as `(1, NULL)` or `(NULL, NULL)`. This bug has existed since v20.2. [#126938][#126938]
+- The `sql_sequence_cached_node` value of the [`serial_normalization` setting]({% link v24.1/serial.md %}#modes-of-operation) was not correctly formatted. This could lead to errors while connecting to CockroachDB if the default value of `serial_normalization` was set to `serial_normalization`. The formatting bug was fixed, which also fixes the errors when connecting. [#127675][#127675]
+
+<h3 id="v24-1-3-performance-improvements">Performance improvements</h3>
+
+- Starting a `cockroach` process will no longer flush buffered [logs]({% link v24.1/logging-overview.md %}) to configured [logging sinks]({% link v24.1/configure-logs.md %}#configure-log-sinks) unless the process is running under `systemd`, where cockroach runs with the `NOTIFY_SOCKET` environment variable. [#126305][#126305]
+- [Schema changes]({% link v24.1/online-schema-changes.md %}) that cause a data backfill, such as adding a non-nullable column or changing the primary key, will now split and scatter the temporary indexes used to perform the change. This reduces the chance of causing a write hotspot that can slow down foreground traffic. [#126684][#126684]
```

Comment:
'write hotspot' could link to https://www.cockroachlabs.com/docs/v24.1/performance-best-practices-overview#hot-spots